### DISCUSSION TITLE
How to update Open WebUI?

### DISCUSSION THREAD

**Discussion by egypsiano** [community-scripts/ProxmoxVE#949] [Q&A] [answered]
I couldn't find a way to update Open WebUI
anyway to do that?
thank you

**Comment by michelroegl-brunner** [comment]
In the lxc shell use `update` or simply run the install comand inside the lxc to update.

**Reply by egypsiano** [reply, marked as answer]
this way didn't work for me, but here it's what i did and solve my problem

`cd /opt/open-webui
git pull origin main
npm install
systemctl restart open-webui
journalctl -u open-webui`
 and voila your Open WebUI is updated

**Comment by vulcan4d** [comment]
The update command in the LXC shell worked for me.  Oddly the Proxmox update helper script didn't update this thing after running it manually and also scheduling it.

**Comment by egypsiano** [comment]
I'm terribly sorry, you're right, and It worked for me.
Thank you so much, and was totally my mistake.
`update` command in LXC Console will do the update.

**Comment by konkos1** [comment]
I am currently on v0.6.41 - how do I update to v0.6.43 ?
Simply running update inside the container does not the trick! It is only updating/checking the Ollama installation...

**Reply by MickLesk** [reply]
Is there uv package. Ask there, they dont have published it

**Reply by konkos1** [reply]
What do you mean with that?

**Reply by MickLesk** [reply]
its an uv package. if they dont update it, you dont get updates

**Reply by konkos1** [reply]
Sorry for my ignorance, but where can I find the uv package for Open WebUI?

**Reply by MickLesk** [reply]
Ask at openwebui Repo?

**Reply by konkos1** [reply]
I've found the "workaround" here: [https://github.com/open-webui/open-webui/issues/20117](url)

**Reply by eudysanchez** [reply]
I had the same issue, I installed open-webui using the proxmox script but i wanted to update to the latest. After many hours researching online, i kinda put the pieces together: Open-webui is installed on an uv environment as stated above, so we need to update the installation environment. The command worked for me was:
 `uv pip install --upgrade open-webui`

**Reply by MickLesk** [reply]
https://github.com/community-scripts/ProxmoxVE/blob/main/ct%2Fopenwebui.sh#L128

......

**Reply by jackchoui** [reply]
i am also in on v0.6.41. Non of the above solutions worked for me to update. What is the good way to update thats future proof?

**Reply by tremor021** [reply]
I'm closing this now as this has nothing to do with our script. this is upstream issue thats already been resolved and will be fixed on the next release.
